### PR TITLE
OAuth scopes for integration tests

### PR DESCRIPTION
**Pull request by dpisarenko**
I added the information about OAuth scopes for various integration tests according to https://developer.github.com/v3/oauth/#scopes , but I'm not 100 % sure that they are correct.

I couldn't figure out what scope `RtGitignoresITCase` needs.

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks for the pull request, let me find a reviewer..

**Comment by dmarkov** [comment]
@krzyk please review this PR

File: src/test/java/com/jcabi/github/package-info.java [hunk 1]
```diff
@@ -30,7 +30,47 @@
 
 /**
  * Object Oriented Github API, tests.
- *
+ * <p>The individual integration tests require following <a href=
```
Comment (krzyk) [inline src/test/java/com/jcabi/github/package-info.java]: let's remove those from package-info

**Comment by krzyk** [comment]
@dpisarenko based on discussion in the issue, you can remove the changes in package-info, and add information in the README.md - but only about all the scopes (not per test) needed to execute integration tests.

**Comment by krzyk** [comment]
@dpisarenko please close this issue if you are not working on this

**Comment by dmarkov** [comment]
@krzyk thank you, added **16 mins** to your acc, payment num is `52798273`... +16 to your rating, your total score is [+4674](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**